T012_M1_load_data.py

```diff
@@ -48,7 +51,7 @@
 
 
 
-def student_age_dictionary(file_name: str) -> dict:
+def student_age_dictionary(file_name: str) -> dict[list]:
```

Comment:
Not a fan of this: Concerned about possible errors being thrown as a result of some kind of conflict here. There's a reason the "typing" module exists imo.